roles/wildfly_validation/tasks/pidfile_validate.yml

```diff
@@ -8,15 +8,18 @@
   when:
     - wildfly_validation_systemd_pidfile is defined and wildfly_validation_systemd_pidfile | length > 0
 
+- name: "Get {{ wildfly_instance_name }} process ID"
+  ansible.builtin.command: "pgrep -f {{ wildfly_instance_name }}"
+  register: wildfly_pid
+  ignore_errors: yes
```

Comment:
So, here is my problem. I don't like that we use pgrep (or ps) instead of Ansible module. However, the pids module has been moved to general, so we can't use it, but if we use pgrep or ps, we then need to add a package: tasks to ensure that the command is available... it's becoming a whole thing, just to check that the pid in the service matches the one running.

I think checking that the pidfile exists, is not empty and contain a valid pid number is enough. Especially knowing, that before we get here, we already established that the systemd service is running.

Reply:
there is another way as well. using systemd module will register the service then using stat we can get the pidfile value and we can use it to compare.
else we can go back to the previous version before my pidfile fix. I'll remove these.

Reply:
or if you really really want to do it, check `/proc/<slurped_pid>` exists (but what happens if procps is not installed or /proc is not mounted)?

Reply:
then it's best to remove this pidfile validation part, we are already checking the file path no need to carry the extra burden of checking the content inside it.

Reply:
changed it back to how it was earlier @rpelisse @guidograzioli